core/node/infra/metrics_publisher.go

```diff
@@ -13,25 +13,26 @@ import (
 	"github.com/river-build/river/core/node/dlog"
 )
 
-var DefaultDurationBucketsSeconds = []float64{
-	0.001,
-	0.002,
-	0.005,
+var DefaultRpcDurationBucketsSeconds = []float64{
```

Comment:
@sergekh the existing bucket setting looks like it was used for two types of metrics:
- things that measure rpc calls, and
- measurement of db transactions.

Each bucket has been updated to better reflect what I was seeing in omega on datadog for both types.